Extension/src/LanguageServer/client.ts

```diff
@@ -705,12 +705,11 @@ class DefaultClient implements Client {
     }
 
     private updateInactiveRegions(params: InactiveRegionParams): void {
-        let renderOptions: vscode.DecorationRenderOptions = {
-            light: { color: "rgba(175,175,175,1.0)" },
-            dark: { color: "rgba(155,155,155,1.0)" },
-            rangeBehavior: vscode.DecorationRangeBehavior.ClosedOpen
-        };
-        let decoration: vscode.TextEditorDecorationType = vscode.window.createTextEditorDecorationType(renderOptions);
+        let settings: CppSettings = new CppSettings(this.RootUri);
+        
+        let decoration: vscode.TextEditorDecorationType = vscode.window.createTextEditorDecorationType({
+            opacity: String(settings.inactiveRegionOpacity)
```

Comment:
you shouldn't need to make this a string since you already enforced the inactiveRegionOpacity setting to be a string in settings.ts.

Reply:
```opacity``` is a ```S```tring, not a ```s```tring, unfortunately. Is it okay to do the conversion here?